src/directives/drawfeature.js

```diff
@@ -11,14 +11,14 @@ goog.require('ngeo.drawpointDirective');
 goog.require('ngeo.drawrectangleDirective');
 /** @suppress {extraRequire} */
 goog.require('ngeo.drawtextDirective');
-/** @suppress {extraRequire} */
-goog.require('ngeo.measureareaDirective');
-/** @suppress {extraRequire} */
-goog.require('ngeo.measureazimutDirective');
-/** @suppress {extraRequire} */
-goog.require('ngeo.measurelengthDirective');
 goog.require('ol.Feature');
 
+/** @suppress {extraRequire} */
+goog.require('ngeo.measure.module');
```

Comment:
That is not good. Directive code should not require a toplevel module.
Only applications and examples require top level modules. Invidivual Angular files must require the fine grained dependency they actually need.

This principle is necessary to avoid bloat. We can discuss if you think there are cases when it does not make sense.

Reply:
No you're right.
(Except eventually for some GMF directive that will be moved soon. But that's not the case here.)